apps/percona/deployers/percona.yml

```diff
@@ -34,6 +34,12 @@ spec:
           volumeMounts:
             - mountPath: /var/lib/mysql
               name: data-vol
+          livenessProbe:
+            exec:
+              command: ["bash", "sql-test.sh"]
+            initialDelaySeconds: 60
```

Comment:
this value should be good. But, thinking whether there is a way to avoid this hard-coded value which can depend on the utilization or H/W specifications of node, something like readiness (or) some checks in sql-test.sh itself to find whether the pod is ready or not.

Reply:
Good idea @vishnuitta ! readiness probe is the way to go.  Can I work on this in a separate PR ? 

The sql-test.sh itself has a DB init wait period (which is also hard-coded, btw) before it starts  the periodic mysql `create table-insert data-drop db` routine - which makes it ~660s of max period for DB init. 

```
echo -e "\nWaiting for mysql server to start accepting connections.."
retries=10;wait_retry=60
for i in `seq 1 $retries`; do
  mysql -uroot -pk8sDem0 -e 'status' > /dev/null 2>&1
  rc=$?
  [ $rc -eq 0 ] && break
  sleep $wait_retry
done
```